src/tiled/scriptdialog.cpp

```diff
@@ -267,6 +291,20 @@ ScriptDialog::NewRowMode ScriptDialog::newRowMode() const
     return m_newRowMode;
 }
 
+ScriptButtonGroup *ScriptDialog::addRadioButtonGroup(const QString &labelText,
+                                                     const QStringList &values,
+                                                     const QString &toolTip,
+                                                     const QStringList &buttonToolTips)
+{
+    QGroupBox *groupParent = new QGroupBox(this);
+    QHBoxLayout *hBox = new QHBoxLayout(groupParent);
+    ScriptButtonGroup *buttonGroup = new ScriptButtonGroup(groupParent, hBox);
+    buttonGroup->addItems(values, buttonToolTips);
+    hBox->addStretch(rightColumnStretch);
```

Comment:
Hmm, this stretch will sit after the items added through this function, and before the items added using `addItems`/`addItem` on the returned `ScriptButtonGroup`, right?

Reply:
Yeah, I think this is unnecessary. I was probably  just trying to figure out why the buttons weren't showing up and messing with the layout. I removed it